### PR TITLE
Update example with o-header@11 

### PR DESCRIPTION
**Pull request by notlee**
Depends on some o-header tsx template fixes.

### CODE REVIEW

**Comment by notlee** [comment]
Depends on: https://github.com/Financial-Times/origami/pull/960

**Comment by notlee** [comment]
Person 1: Hey Lee, why don't we get rid of Handlebars now? There's no reason to use a mix of TSX and Handlebars, it's not like no o-footer TSX is a dependency of switching. 

Lee: I know, right! Putting a little tech debt on the credit card here 😅 I think I'd prefer to focus on our current objectives and schedule this work separately, perhaps along with a bigger focus on services – should we move away from Handlebars across the board?

File: src/example-header.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      "subsections-right": [],
+      "user": navigationData['user']
+    },
+    "variant": "simple",
+    "showMegaNav": false,
+    "userIsSubscribed": false,
+    "showLogoLink": true
+  }
+
+console.log(Server.renderToString(
```
Comment (frshwtr) [inline src/example-header.tsx:79]: Is the log needed?

Comment (notlee) [reply src/example-header.tsx:79]: Yeah that one is needed. During `npm run build` this outputs the rendered html to standard output which is then written to a file using `tee`.

**Pull request merged**